src/base/OpenMCCellAverageProblem.C

```diff
@@ -3063,6 +3085,11 @@ OpenMCCellAverageProblem::syncSolutions(ExternalProblem::Direction direction)
         if (_volume_calc)
           _volume_calc->resetVolumeCalculation();
 
+        if (_use_displaced)
```

Comment:
let's figure out just the single place where this call needs to go -- we shouldn't need it more than once in this class.  I think this is the right place to have it. Can you try one-by-one progressively deleting, re-compiling, and re-running your test case in order to confirm we can have just this single call?